spec/features/user_visits_place_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.feature 'following a place' do
+RSpec.feature 'visiting a place' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.